reference/release-notes/1.26.0.md

```diff
@@ -0,0 +1,116 @@
+
+## New features & improvements
+
+### Deployment and Operations
+
+* You can now deploy Anbox Cloud using a reference Terraform plan. See {ref}`howto-deploy-anbox-terraform` for detailed instructions.
+* To allow open source charms better integrate with our charms, we no longer block the installation of snaps within the charms if a UA token in not attached. The deployment will proceed with a warning instead of failing. However, remember that without attaching a UA token, you won't be able to access images when the deployment is complete.<!--AC-3308-->
+* Service restarts now ask for your confirmation, unless you are using non-interactive mode. If you are using non-interactive mode, use the `--no-restart` flag to disable automatic restart of AMS and other required services.<!--AC-3283-->
+* Appliance initialization now gives clearer guidance on acceptable block devices.<!--AC-3321-->
+
+### Instance management
+
+* You can now specify your GPU type when launching an instance by using the `--gpu-type` flag (Possible values:`amd`, `intel`, `nvidia`).<!--AC-3259-->
+* Default version for NVIDIA driver is 570.<!--AC-3246-->
+* AMS can now automatically determine if the instance is based on an application or an image. So you don't have to use `--raw` every time you launch an instance based on an image.<!--AC-3192-->
+
+### Logging
+
+* The Anbox log level can be changed at runtime using the [Anbox HTTP API](https://documentation.ubuntu.com/anbox-cloud/reference/api-reference/anbox-https-api/). This is helpful in situations when you want to avoid a restart of the Anbox runtime.<!--AC-3183-->
+
+### Dashboard enhancements
+
+We have the following enhancements to the Anbox Cloud dashboard:
+
+* Overhaul of the registry page
+* Customization of instance resources including CPU, GPU, memory
+* Authentication using Keycloak and Hydra. See {ref}`howto-appliance-enable-oidc` for setup instructions.
+* Ability to revoke of a session share via ADB and also extend the validity of a share.
+* Notifications via websockets providing a reliable way to notify about asynchronous operations.
+
+### Images
+
+* VM images now use hardware enablement kernels.<!--AC-3304-->
+* With this release, images for Ubuntu 24.04 (Noble Numbat) and Android 15 (AOSP and AAOS) are available.
+* Unused packages including `netcat-openbsd`, `xxd`, `lbzip2`, `cron` are removed from AOSP images.<!--AC-3158-->
+
+### Streaming
+
+* We have upgraded to use the WebRTC revision 7049.<!--AC-3310-->
+* You can now explicitly disable certain video codecs (AV1, H264, VP8) even if they are supported by the client. Use the `video.disabled_codecs` option in the WebRTC streamer configuration file.<!--AC-3314-->
+
+### Other
+
+* Android security updates for May 2025 (see [Android Security Bulletin - May 2025](https://source.android.com/docs/security/bulletin/2025-05-01)).<!--AC-3176-->
+* The Android WebView has been updated to 136.0.7103.60.<!--AC-3316-->
+
+## Deprecations
+
```

Comment:
If we agree on deprecating the node controller charm, we need to add it to this section.